modules/node_modules/@ncigdc/containers/FrequentlyMutatedGenesTable.js

```diff
@@ -145,6 +146,11 @@ const FrequentlyMutatedGenesTableComponent = compose(
             ),
           },
           {
+            key: 'annotations',
+            title: 'Annotations',
+            style: { padding: '5px 0 0 0', textAlign: 'center' },
```

Comment:
wishlist for tables (unrelated to PR) @alex-wilmer: 
something like `headingStyle`, `cellStyle`